corehq/apps/zapier/signals/receivers.py

```diff
@@ -0,0 +1,13 @@
+from django.dispatch import receiver
+
+from corehq.apps.zapier.tasks import send_to_subscribers_task
+from corehq.toggles import ZAPIER_INTEGRATION
+from couchforms.signals import successful_form_received
+
+
+@receiver(successful_form_received)
+def send_form_subscribers(sender, xform, *args, **kwargs):
+    domain = xform.domain
+    if not ZAPIER_INTEGRATION.enabled(domain):
+        return
+    send_to_subscribers_task.delay(domain, xform)
```

Comment:
I think in most other parts of the codebase we just send hte ID to celery and get the document again from the database. i'm not sure on whether sending the whole document can be a burden on rabbitmt... @gcapalbo might know better